agents/rust/mediator/src/didcomm_handlers/mod.rs

```diff
@@ -47,13 +46,21 @@ pub async fn handle_aries<T: BaseWallet + 'static, P: MediatorPersistence>(
             handle_routing_forward(agent.clone(), forward).await?;
             return Ok(Json(json!({})));
         } else {
-            let (account_name, our_signing_key, their_diddoc) =
+            // Auth known VerKey then process account related messages
```

Comment:
this seems like a "todo", consider prefixing the comment text with `// todo: `, it tends to have different highlighting in IDEs

Reply:
This one is just a comment. The `auth_and_get_details` method does the auth. 
Just had the comment to explain the latter flow. 

Thanks for tip about `// todo:`